vita-cera/app/r2purs/Main.hs

```diff
@@ -27,6 +27,11 @@ import Language.PureScript.Bridge.TypeParameters
 
 import Renaissance.Api.Bz
 
+tokenBridge :: BridgePart
```

Comment:
The token bridge is necessary because Token is not encode in Json using the by default implementation. Therefore we need to bypass the type and the Purescript implementation wil think it is just a String